index.html

```diff
@@ -14,29 +14,18 @@
 	<link href="css/achilles.css" rel="stylesheet">
 	<link href="css/chart.css" rel="stylesheet">
 
-	<script>
-		var report = 'dashboard';
-		var page_vm = {};
-
-		function updateReport(value) {
-			report = value;
-			updateRoute();
-		}
-
-		function setDatasource(index) {
-			page_vm.datasource(page_vm.datasources[index]);
-			updateRoute();
-		}
-
-		function updateRoute() {
-			$('.reportDrilldown').addClass('hidden');
-			document.location = '#/' + page_vm.datasource().folder + '/' + report;
-		}
-	</script>
-
 	<script src="js/curl.config.js"></script>
 	<script src="js/curl.js"></script>
+	<script src="js/jquery-1.10.2.js"></script>
 	<script src="js/app/achilles.js"></script>
+	<script src="js/app/achillesWrapper.js"></script>	
+	<script>
+		var datasourcepath = 'data/datasources.json'
+		var containerName = "#achillesContainer";
+		var achillespath = "achilles.html";
+		var a = new Achilles( datasourcepath, containerName, achillespath, achillesWeb);
```

Comment:
We changed the code in order to have a AchillesWrapper that depends on achilles.js. In that way, achilles will be loaded by AMD. Or do you have other suggestion?